.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+go:
+  - 1.4
+  - 1.5
+install:
+  - go get golang.org/x/tools/cmd/vet
+script:
+  - cd $TRAVIS_BUILD_DIR/multi-node/aws
```

Comment:
Why not test the entire repo? Currently the only go code is `multi-node/aws`, but it would be good to catch other code in the future if/when it's added.